frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -821,6 +823,22 @@ class DataApi {
     return Store.getState().temporaryConfiguration.activeMapping.isMappingEnabled;
   }
 
+  refreshIsosurfaces(): Promise<void> {
+    Store.dispatch(refreshIsosurfacesAction());
+    const isRefreshingComplete = new Promise(resolve => {
+      const unsubscribe = listenToStoreProperty(
+        state => state.uiInformation.refreshingIsosurfaces,
+        refreshingIsosurfaces => {
+          if (!refreshingIsosurfaces) {
+            unsubscribe();
+            resolve();
+          }
+        },
+      );
+    });
+    return isRefreshingComplete;
+  }
```

Comment:
I could't come up with a better solution for this (to wait for all isosurface to be refeshed). Do you think thats fine @philippotto?